protocol/proto/openyolo.proto

```diff
@@ -21,6 +21,7 @@ package org.openyolo;
 option java_package = "org.openyolo.protocol";
 option java_outer_classname = "Protobufs";
 
+// Next 6
```

Comment:
I'm not sure how I feel about these types of comments; it's another thing to update, and maintainers of protos should be careful enough to increment properly. With protov3, we can ensure that every field ID is accounted for via deprecation and reserved field declarations.

Reply:
Good point, removed.